internal/examples/protobuf-hello/client/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	hello "go.uber.org/yarpc/internal/examples/protobuf-hello"
+	"go.uber.org/yarpc/transport/http"
+	"go.uber.org/yarpc/x/config"
+)
+
+const yaml = `
+outbounds:
+  server:
+    http:
+      url: "http://localhost:8080/"
+`
+
+func main() {
```

Comment:
I again register my distaste for this style, would prefer:

```go
func main() {
  if err := do(); err != nil {
    log.Fatal(err)
  }
}
```

And then don't use panic functions. I think this style being in the example sets a good example (get it?).